index.html

```diff
@@ -33,13 +39,128 @@
                   ? `https://github.com/prefuse/Flare/tree/master/flare/src/${n.ancestors().reverse().map(d => d.data.name).join("/")}`
                   : `https://github.com/prefuse/Flare/blob/master/flare/src/${n.ancestors().reverse().map(d => d.data.name).join("/")}.as`,
                 */
-                width: 1152,
-                height: 1152
+                width: mainWidth,
+                height: mainHeight
             });
 
             document.querySelector("#sunburstContainer")
                 .appendChild(chart);
-        });
+
+            
+            //PNG EXPORT FUNCTIONALITY
+            // Set-up the export button
+            d3.select('#saveButton').on('click', function(){
+              let svgString = getSVGString(document.getElementById('theImage'));
+              
+              svgString2Image( svgString, 2*mainWidth, 2*mainHeight, 'png', save ); // passes Blob and filesize String to the callback
+
+              function save( dataBlob, filesize ){
+                saveAs( dataBlob, 'D3 vis exported to PNG.png' ); // FileSaver.js function
```

Comment:
give it a better file name, e.g. sunburst-smorgasbord.png; later we can even label it differently when people give it their own title